frontend/src/features/admin-form/common/components/AdminFormNavbar/AdminFormNavbarContainer.tsx

```diff
@@ -25,7 +25,7 @@ const useAdminFormNavbar = () => {
   const navigate = useNavigate()
 
   const calcCurrentIndex = useCallback(() => {
-    const index = ADMINFORM_ROUTES.findIndex((r) => r && pathname.endsWith(r))
+    const index = ADMINFORM_ROUTES.findIndex((r) => r && pathname.includes(r))
```

Comment:
might be worth including a comment here stating why the truthy check for `r` is necessary (because the index route is an empty string which will always return true for `includes`)